src/test/java/org/waarp/openr66/dao/database/mariaDB/DBBusinessMariaDBDAOIT.java

```diff
@@ -1,21 +1,21 @@
-package org.waarp.openr66.dao.database.mariaDB.test;
+package org.waarp.openr66.dao.database.mariaDB;
```

Comment:
Usually we keep the package name as in the main, so `mariadb`, not `mariaDB` . Changing the package (and directory) name could be a good idea.

Same for other tests related to other databases